api/src/test/java/uk/gov/dhsc/htbhf/claimant/controller/ClaimControllerIntegrationTest.java

```diff
@@ -0,0 +1,64 @@
+package uk.gov.dhsc.htbhf.claimant.controller;
+
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import uk.gov.dhsc.htbhf.claimant.entity.Claimant;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimDTO;
+import uk.gov.dhsc.htbhf.claimant.repository.ClaimantRepository;
+import uk.gov.dhsc.htbhf.claimant.testsupport.ClaimDTOTestDataFactory;
+
+import javax.transaction.Transactional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@SpringBootTest
+@Transactional
+public class ClaimControllerIntegrationTest {
+
+    @Autowired
+    ClaimController controller;
+
+    @Autowired
+    ClaimantRepository claimantRepository;
+
+    @Test
+    void shouldPersistNewClaimant() {
```

Comment:
Would probably be clearer called `shouldPersistNewClaimantWithoutExpectedDeliveryDate`.